src/entities/candidate/candidate.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Param, ParseIntPipe, Req } from '@nestjs/common';
```

Comment:
Remove `Req`  import since it wasn't used in the controller methods.

Reply:
Done